app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java

```diff
@@ -2237,6 +2247,17 @@ public void reject() {
     @Override
     public boolean isEyeTrackingSupported() { return mIsEyeTrackingSupported; }
 
+    @Keep
+    @SuppressWarnings("unused")
+    private void changeWindowDistance(float aDelta) {
+        // First we need to adapt the range of movement of the hand/controller to the actual range
+        // of movement of the window in space. We use 40cm for arm length
+        float windowWorldZRange = Math.abs(WidgetPlacement.floatDimension(getApplicationContext(), R.dimen.window_world_z_max) - WidgetPlacement.floatDimension(getApplicationContext(), R.dimen.window_world_z_min));
+        float conversionRatio = windowWorldZRange / 0.4f;
+        float clamped = Math.max(0.0f, Math.min(mSettings.getWindowDistance() + aDelta * conversionRatio, 1.0f));
```

Comment:
How the result of the Math.min inside could be negative ?

Reply:
Because delta is negative if you approach the controller